index.html

```diff
@@ -316,12 +316,14 @@ <h2 class="section-title">Venue</h2>
 <!-- ## SPONSOR SECTION ## -->
 	<section class="sponsors-section qcContainer" id="sponsors">
 	  <div class="qcTabTitle clearfix">
-			<div class="col-4 col">
-	      <h2 class="section-title">Sponsors</h2>
+			<div class="row middle-xs">
+				<div class="col-xs-12 col-sm-6">
+					<h2 class="section-title">Sponsors</h2>
+				</div>
+				<div class="col-xs-12 col-sm-6 section-button-cta">
+					<a href="assets/sponsor-prospectus.pdf" target="_blank" class="button secondary">Interested in becoming a sponsor?</a>
+				</div>
```

Comment:
In the original design, the grid system hides the the column on the right on smaller screens. Here I changed the columns here to the flexboxgrid classes so that the "Interested in becoming a sponsor" button was visible at all times since I thought that was easier than changing the theme css.